bin/test-stub

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# need bash, not sh, for default variable ${var:-default}
+
+exercisedir=$1
+exercisename=$(basename "$exercisedir")
+examplename="stub"
+
+buildfolder="${TRAVIS_BUILD_DIR:-.}/build/${exercisename}/${examplename}"
+mkdir -p "${buildfolder}"
+cp -rL ${exercisedir}/stack.yaml ${exercisedir}/package.yaml ${exercisedir}/src ${exercisedir}/test "${buildfolder}"
+
+cd $buildfolder
+
+examplecache="${HOME}/.foldercache/${exercisename}/${examplename}/.stack-work"
```

Comment:
OK to create a .foldercache in user's personal HOME if running locally? If not, how do we deal with this? `if [ -n "$TRAVIS" ] examplecache="something-else"` perhaps?